src/sdk/namespace_fs.js

```diff
@@ -1294,10 +1292,8 @@ class NamespaceFS {
         let stat = await target_file.stat(fs_context);
         this._verify_encryption(params.encryption, this._get_encryption_info(stat));
 
-        // handle xattr
-        // assign user xattr on non copy / copy with xattr_copy header provided
         const copy_xattr = params.copy_source && params.xattr_copy;
-        let fs_xattr = copy_xattr ? undefined : to_fs_xattr(params.xattr);
+        let fs_xattr = to_fs_xattr(params.xattr);
```

Comment:
on copy xattr you are supposed to also set the new xatter together with the copied ones?

Reply:
yes, for example we need to set new version-id, what xattr needs to be copied should be determined by fix_copy_source_params:
`  if (params.xattr) params.xattr = _.omitBy(params.xattr, (val, name) => name.startsWith?.('noobaa-namespace'));`

Reply:
right, but you meant to refer to params.xattr = source_md.xattr which does this copy in fix_copy_source_params.

but don't we have the same case to fix also in _create_empty_dir_content ?
which seems to mean that _get_copy_source_xattr() is unneeded completely?